docs/design/mono/diagnostics-tracing.md

```diff
@@ -95,17 +95,13 @@ On the other hand, if it is desired to include all components, there are two opt
             <UseAllRuntimeComponents>true</UseAllRuntimeComponents>
         </PropertyGroup>
         ```
+
 ## Install diagnostic client tooling
 
 ```
```

Comment:
nit: code blocks in this doc can use language tag to enable syntax highlighting.

````suggestion
```sh
````